jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -149,20 +149,36 @@ ecma_is_value_boolean (ecma_value_t value) /**< ecma-value */
  *         false - otherwise.
  */
 bool __attr_pure___ __attr_always_inline___
-ecma_is_value_true (ecma_value_t value) /**< ecma-value */
+ecma_is_value_true (ecma_value_t value) /**< ecma value */
 {
   return (ecma_get_value_type_field (value) == ECMA_TYPE_SIMPLE
           && ecma_get_value_value_field (value) == ECMA_SIMPLE_VALUE_TRUE);
 } /* ecma_is_value_true */
 
 /**
+ * Check if the value is false.
+ *
+ * Warning:
+ *         value must be boolean
```

Comment:
To tell the truth I just copied&pasted the description from above (see: `ecma_is_value_true`). We can remove that warning message, it looks useless to me too.